metro.config.windows.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+const fs = require('fs');
+const path = require('path');
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+const rnPath = fs.realpathSync(
+  path.resolve(require.resolve('react-native/package.json'), '..'),
+);
+const rnwPath = fs.realpathSync(
+  path.resolve(require.resolve('react-native-windows/package.json'), '..'),
+);
+
+module.exports = {
+  resolver: {
+    extraNodeModules: {
+      // Redirect react-native to react-native-windows
+      'react-native': rnwPath,
+      'react-native-windows': rnwPath,
+    },
+    // Include the macos platform in addition to the defaults because the fork includes macos, but doesn't declare it
+    platforms: ['ios', 'android', 'windesktop', 'windows', 'web', 'macos'],
+    // Since there are multiple copies of react-native, we need to ensure that metro only sees one of them
+    // This should go in RN 0.61 when haste is removed
```

Comment:
// This should go in RN 0.61 when haste is removed

is this relevant now? Should it be removed / edited..?

Reply:
I can change the number to 62, but we still need to override the metro config for out-of-tree platforms. There is an active PR to make it unnecessary
https://github.com/react-native-community/cli/pull/1115
(partial fix) https://github.com/microsoft/react-native-windows/issues/4081